IP-POLICY-GUIDANCE.md

```diff
@@ -0,0 +1,190 @@
+Please reach out to the Foundation with any questions on topics related to the
+IP Policy by emailing
+[legal-questions@lists.openjsf.org](mailto:legal-questions@lists.openjsf.org).
+
+## Copyrights and OpenJS Foundation projects
+
+### Ownership of Copyrights in OpenJS Foundation Contributions
+
+When source code, documentation and other content is contributed to an Open JS Foundation
+project, the copyrights in those contributions remain owned by the original
+copyright holders.
+
+The copyrights are not _assigned_ to OpenJS Foundation. Instead, they are _licensed_ for
+distribution as part of the project. Whether a project uses the <abbr title="Developer Certificate of Origin">[DCO](https://developercertificate.org)</abbr> or the <abbr title="Contributor License Agreement">[OpenJS Foundation CLA](https://cla.openjsf.org)</abbr>,
+the original copyright holders retain their copyrights.
+
+### Copyright Notices
+
+OpenJS Foundation does not require or recommend that every contributor include their
+copyright notice in contributed files. [See below for more details on why
+not.](#why-not-list-every-copyright-holder)
+
+Instead, OpenJS Foundation recommends using a more general statement in a form similar to the
+following (where XYZ is the project's name):
+
+- **Copyright The XYZ Authors.**
+- **Copyright The XYZ Contributors.**
+- **Copyright Contributors to the XYZ project.**
+
+These statements are intended to communicate the following:
+- the work is copyrighted;
+- the contributors of the code licensed it, but retain ownership of their copyrights; and
+- it was licensed for distribution as part of the named project.
+
+By using a common format, the project avoids having to deal with names of
+copyright holders, years or ranges of years, and variations on the (c) symbol.
+This aims to minimize the burden on developers and maintainers as well as
+redistributors of the code.
+
+### What if I want my copyright notice included?
+
+Please note that it is _not wrong_, and it is acceptable, if a contributor
+wishes to keep their own copyright notices on their contributions. The above is
+a recommended format for ease of use, but is not mandated by OpenJS Foundation.
+
+If you are contributing on behalf of your employer, you may wish to discuss with
+your legal department about whether they will require you to include a copyright
+notice identifying them as the copyright holder in contributions.
+
+### What about Third Party Code?
+
+If a file only contains code that originates from a third party source who
+didn't contribute it themselves, then you would _not_ want to add the notices
+above. (In a similar vein, you wouldn't add a notice identifying you as the
+copyright holder either, if you didn't own it.) Just preserve the existing
+copyright and license notices as they are.
+
+If, however, you add copyrightable content to a pre-existing file from another
+project, then at that point you could _add_ a copyright notice similar to the
+one above.
```

Comment:
It would be nice to have a bit of guidance on projects that are forks of others.

Reply:
@mcollina could you maybe expand on this a little?  At first glance I don't think there would be a difference, but I could easily be missing a subtlety, too.

Reply:
The reason is that when onboarding the copyright is attached to the previous contributors. Should that stay the same and just add one more line?

Reply:
Ah, yeah. So this would fall under the blanket guidance at the beginning, to never change an existing copyright line unless you have the rights to do so.  In most cases, the only time you have the rights are if your employer asks you to update a line that refers to them, or if you put it there on your own behalf.

Reply:
I think we should clarify _could_.